### PR TITLE
CMS Integration

### PR DESCRIPTION
**Pull request by rihp**
This iteration of the site is an exact copy of the current site, but the majority of content is fetched from Contentful API.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+REACT_APP_CMS_SITE=`https://graphql.contentful.com/content/v1/[SITEHERE]/`
+REACT_APP_CMS_TOKEN=Bearer [TOKENHERE]
+MYVAR=FOO
```
Comment (dOrgJelli) [inline .env.example:3]: Why is this here?

File: src/components/CMScontent.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            }  
+            docsLink
+          }
+        }
+      }`
+
+    return ContentfulFetcher(cmsQuery)
+
+
+    /* Sample of what the returned data would look like!
```
Comment (dOrgJelli) [inline src/components/CMScontent.tsx:31]: Let's remove commented out code

File: src/components/CMScontent.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+        // we immediately iterate on the QUERIES that the wrappers have
+        wrapper.queriesCollection.items.forEach((item) => {
+            
+            // Check if this is a featured QUERY or not
+            if (item.featured){
+                
+                // If it is, add the relevant data to the array `newWrappersList`
+                // idea:  modify this data stucture below to fit all the needs Wrappers.tsx
+                // listOfFeaturedQueries.push({
```
Comment (dOrgJelli) [inline src/components/CMScontent.tsx:130]: here as well

File: src/components/CMScontent.tsx [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+            }
+        });
+    });
+    
+    // This would return an array with the organized QUERIES
+    return listOfFeaturedQueries
+}
+
+// export function queryFeaturedQueries() {
```
Comment (dOrgJelli) [inline src/components/CMScontent.tsx:176]: here as well

File: src/components/Features.tsx [hunk 5]
```diff
@@ -74,43 +76,94 @@ const useStyles = makeStyles((theme) => ({
...
+
 
 export const Features = () => {
   const theme = useTheme();
   const classes = useStyles();
 
+
+  //var counter: number = 0;
+  
+  // TODO : Hooks ?
```
Comment (dOrgJelli) [inline src/components/Features.tsx:95]: hooks?

File: src/components/Hero.tsx [hunk 6]
```diff
@@ -10,6 +12,9 @@ import {
 import { makeStyles } from '@material-ui/core/styles';
 import KeyboardArrowRightOutlined from '@material-ui/icons/KeyboardArrowRightOutlined';
 import { polywrapPalette } from '../theme';
+// TODO: should we deprecate the verbiage folder?
+//import { CTA } from '../constants/verbiage';
```
Comment (dOrgJelli) [inline src/components/Hero.tsx:16]: remove commented out code

File: src/components/HubCallout.tsx [hunk 7]
```diff
@@ -1,6 +1,15 @@
 import { Parallax } from 'react-scroll-parallax';
 import { Box, Grid, makeStyles, Typography, useMediaQuery, useTheme } from '@material-ui/core';
 import { polywrapPalette } from '../theme';
+// WIP: Try to modularize the CMS query
+import {useState, useEffect} from 'react';
+import {  webContent, ContentfulFetcher } from './QueryModule';
+
+//These imports are for multiple slides/caroussel effect
```
Comment (dOrgJelli) [inline src/components/HubCallout.tsx:8]: remove commented out code

File: src/components/Features.tsx [hunk 8]
```diff
@@ -74,43 +76,94 @@ const useStyles = makeStyles((theme) => ({
...
-  },
-  {
-    slug: 'upgradable',
-    title: 'Upgradable',
-    description: 'Ditch the rebuilds with configurable run-time updates.',
-  },
-]
+// CONTENTFUL CMS  INITIAL SET UP BELOW       // v FEATURES v
+
+// TODO : Iterate through myFeatures, and perform all the queries respectively
```
[the remark this answers is not in the document]
Comment (rihp) [reply src/components/Features.tsx]: yes, removed.

File: .env.example (hunk 1 above)
Comment (rihp) [reply .env.example:3]: I've been told before it's good to have the `.env.example` as a template for other devs to set up the actual `.env` locally.

Comment (dOrgJelli) [reply .env.example:3]: I was referencing the `MYVAR=FOO`

Comment (rihp) [reply .env.example]: it's not needed i'll take it out

**Comment by rihp** [comment]
These changes are included in this more complete and cleaner PR https://github.com/polywrap/landing-page/pull/136
